api-server/main.go

```diff
@@ -214,7 +214,7 @@ func (srv *ILabServer) runServer(cmd *cobra.Command, args []string) {
 		srv.ilabCmd = ilabPath
 	} else {
 		// Use ilab from virtual environment
-		srv.ilabCmd = filepath.Join(srv.baseDir, "venv", "bin", "ilab")
+		srv.ilabCmd = filepath.Join(srv.baseDir, "bin", "ilab")
```

Comment:
Does this break the api server running in the default namespace if ilab isn’t copied into PATH? Add a flag if so plx sir.

Reply:
@nerdalert venv is the name of virtual environment, user is to free to choose any name when they create the virtual environment. This installer now pass the entire path <venv-path>/<venv-name> to the apiserver. In this case it would be `<venv-path>/venv`. does that break anything in the apiserver?